src/py/run_generator.rs

```diff
@@ -0,0 +1,290 @@
+use crate::allocator::{Allocator, NodePtr};
+use crate::cost::Cost;
+use crate::gen::conditions::{parse_spends, Condition, SpendConditionSummary};
+use crate::gen::opcodes::{
+    ConditionOpcode, AGG_SIG_ME, AGG_SIG_UNSAFE, ASSERT_HEIGHT_ABSOLUTE, ASSERT_HEIGHT_RELATIVE,
+    ASSERT_SECONDS_ABSOLUTE, ASSERT_SECONDS_RELATIVE, CREATE_COIN, RESERVE_FEE,
+};
+use crate::gen::validation_error::{ErrorCode, ValidationErr};
+use crate::int_to_bytes::u64_to_bytes;
+use crate::py::run_program::OperatorHandlerWithMode;
+use crate::reduction::{EvalErr, Reduction};
+use crate::run_program::{run_program, STRICT_MODE};
+use crate::serialize::node_from_bytes;
+
+use crate::py::f_table::f_lookup_for_hashmap;
+use crate::py::lazy_node::LazyNode;
+
+use std::collections::HashMap;
+use std::rc::Rc;
+
+use pyo3::prelude::*;
+use pyo3::types::{PyBytes, PyDict};
+
+#[derive(Clone)]
+pub struct PyBytesObj {
```

Comment:
You can actually use a real `PyBytes` object here, and it becomes a native python object. There are some examples in `py/run_program.rs` you can look at.

Reply:
I guess if you're only looking at each object once from python, it doesn't matter. But for frequently-used things, you pay a conversion price each time across the bridge unless it's already a python object.

Reply:
I was having a hard time getting that to work